docker/build/scan-urls-repo.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+for i in "$@"; do
+  case $i in
+    -d=*|--directory=*)
+      DIRECTORY="${i#*=}"
+      shift # past argument=value
+      ;;
+    -u=*|--urls_file=*)
+      URLSFILE="${i#*=}"
+      shift # past argument=value
+      ;;
+    -*|--*)
+      echo "Unknown option $i"
+      exit 1
+      ;;
+    *)
+      ;;
+  esac
+done
+
+if test -f "$URLSFILE"; then
+  echo "Scanning directory '$DIRECTORY' from URLs file '$URLSFILE'...";
+  FOUND_URLS=0
```

Comment:
```suggestion
declare -i  FOUND_URLS=0
```

make it a proper integer